community/modules/compute/schedmd-slurm-gcp-v6-nodeset/main.tf

```diff
@@ -52,12 +52,18 @@ locals {
     content  = var.startup_script
   }]
 
+  dws_flex = {
+    enabled = var.dws_flex.enabled
+    max_run_duration = tostring(var.dws_flex.max_run_duration)
```

Comment:
why `tostring` ?

Reply:
I think I had some odd error when using it as a number, in any case the error was unrelated so no need to convert the type.